fastlane/Fastfile

```diff
@@ -121,12 +121,7 @@ before_all do
 end
 
 platform :ios do
-  # Unfortunately, release toolkit still relies on certain settings being set in the environment
-  ENV['PROJECT_NAME'] = 'podcasts'
-  ENV['PROJECT_ROOT_FOLDER'] = "#{PROJECT_ROOT_FOLDER}/"
-  ENV['PUBLIC_CONFIG_FILE'] = VERSION_XCCONFIG_PATH
-  ENV['GHHELPER_REPO'] = GHHELPER_REPO
-
+  # This explicit timeout helps resolve Xcode build errors
```

Comment:
I was curious about this and tested it https://github.com/Automattic/pocket-casts-ios/pull/1187. Got some insights in what it's used for:

![image](https://github.com/Automattic/pocket-casts-ios/assets/1218433/9175ab69-c45b-43eb-a8d9-99bc0a247add)

```suggestion
  # This explicit timeout is necessary for the `xcodebuild -showBuildSettings` call to succeed in CI.
  # See https://buildkite.com/automattic/pocket-casts-ios/builds/4875#018b7eea-e8d7-40a1-8ce5-515dd8feafce/666-805
```